src/javascript/app/pages/trade/contracts.jsx

```diff
@@ -124,7 +124,7 @@ class Contracts extends React.Component {
                     className={`contracts_dropdown ${open ? '' : 'hidden'}`}
                     ref={this.saveRef.bind(null, 'wrapper')}
                 >
-                    <div className='mobile_close invisible'>
+                    <div className={`mobile_close invisible ${open && is_mobile ? '': 'disabled'}`}>
```

Comment:
invisible class never takes effect because of the `!important` [here](https://github.com/zakibinary/binary-static/blob/97e7cfe07891bddc1a47231163a6def1731e4590/src/sass/app/components/contract.scss#L59)
if we remove it, then this can just be:
`<div className={`mobile_close ${open ? '': 'invisible'}`}>`